src/main/java/com/openshift/internal/restclient/DefaultClient.java

```diff
@@ -109,18 +109,29 @@ public IWatcher watch(String namespace, IOpenShiftWatchListener listener, String
 	}
 
 	@Override
+	public IWatcher watch(IOpenShiftWatchListener listener, String...kinds) {
+		return this.watch("", listener, kinds);
```

Comment:
This usecase is a totally legitimete usecase that Can be done with the http api manuelly and with the oc client manually. Why should it not be supported? 

What about creating a PriviledgedClient that has these aditional endpoints that needs priviledge?